packages/plugin-ext/src/hosted/browser/hosted-plugin-manager-client.ts

```diff
@@ -171,12 +173,16 @@ export class HostedPluginManagerClient {
         this.isDebug = true;
 
         await this.start({ debugMode: this.hostedPluginPreferences['hosted-plugin.debugMode'] });
+        const outFiles = this.pluginLocation && [path.join(FileUri.fsPath(this.pluginLocation), '**', '*.js')];
```

Comment:
@benoitf @evidolob  I've noticed that in the plugin ext very often uris on the backend converted to a path via `uri.path.toString()`. It won't work on windows. One should use `FileUri.fsPath`.